### DISCUSSION TITLE
NDEx Notifications - Usage statistics of your public networks - October 2022

### DISCUSSION THREAD

**Discussion by AlexanderPico** [wikipathways/GPML-Repo#99] [Show and tell]
<span style="caret-color: rgb(34, 34, 34); color: rgb(34, 34, 34); font-family: &quot;Trebuchet MS&quot;, &quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, &quot;Lucida Sans&quot;, Tahoma, sans-serif; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: justify; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration: none; display: inline !important; float: none;">Dear wikipathways account holder,</span><br style="caret-color: rgb(34, 34, 34); color: rgb(34, 34, 34); font-family: &quot;Trebuchet MS&quot;, &quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, &quot;Lucida Sans&quot;, Tahoma, sans-serif; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: justify; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><span style="caret-color: rgb(34, 34, 34); color: rgb(34, 34, 34); font-family: &quot;Trebuchet MS&quot;, &quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, &quot;Lucida Sans&quot;, Tahoma, sans-serif; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: justify; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration: none; display: inline !important; float: none;">Thanks for using NDEx to publish your networks. Here are the usage statistics of your public networks in NDEx for the month of October 2022:</span><ul style="caret-color: rgb(34, 34, 34); color: rgb(34, 34, 34); font-family: &quot;Trebuchet MS&quot;, &quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, &quot;Lucida Sans&quot;, Tahoma, sans-serif; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: justify; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><li>Number of downloads this month:<span class="Apple-converted-space"> </span><strong>1530</strong></li><li>Total number of downloads:<span class="Apple-converted-space"> </span><strong>96692</strong></li></ul><span style="caret-color: rgb(34, 34, 34); color: rgb(34, 34, 34); font-family: &quot;Trebuchet MS&quot;, &quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, &quot;Lucida Sans&quot;, Tahoma, sans-serif; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: justify; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration: none; display: inline !important; float: none;">Most downloaded networks in your account:</span><br style="caret-color: rgb(34, 34, 34); color: rgb(34, 34, 34); font-family: &quot;Trebuchet MS&quot;, &quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, &quot;Lucida Sans&quot;, Tahoma, sans-serif; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: justify; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;">

Network Name | Downloads
-- | --
WP534 - Glycolysis and gluconeogenesis - Homo sapiens * | 137
WP4172 - PI3K-Akt signaling pathway - Homo sapiens | 29
Activating invasion and metastasis | 12
WP382 - MAPK signaling pathway - Homo sapiens | 11
WP364 - IL-6 signaling pathway - Homo sapiens | 10
WP4585 - Cancer immunotherapy by PD-1 blockade - Homo sapiens | 10
WP558 - Complement and coagulation cascades - Homo sapiens | 10
WP2203 - Thymic stromal lymphopoietin (TSLP) signaling pathway - Homo sapiens | 10
WP391 - Mitochondrial gene expression - Homo sapiens | 10
P53 pathway | 10
WP1449 - Regulation of toll-like receptor signaling pathway - Homo sapiens | 9
WP4846 - SARS-CoV-2 and COVID-19 pathway - Homo sapiens | 9
WP5095 - Overview of proinflammatory and profibrotic mediators - Homo sapiens | 9
NFKB pathway genes with correlation to HOXA1 expression in breast cancer | 9
WP4754 - IL-18 signaling pathway - Homo sapiens | 9
WP4262 - Breast cancer pathway - Homo sapiens | 9
Tumor promoting inflammation | 8
Therapeutics | 8
WP2855 - Dopaminergic neurogenesis - Homo sapiens | 8
Avoiding immune destruction | 8

<br style="caret-color: rgb(34, 34, 34); color: rgb(34, 34, 34); font-family: &quot;Trebuchet MS&quot;, &quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, &quot;Lucida Sans&quot;, Tahoma, sans-serif; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: justify; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><br style="caret-color: rgb(34, 34, 34); color: rgb(34, 34, 34); font-family: &quot;Trebuchet MS&quot;, &quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, &quot;Lucida Sans&quot;, Tahoma, sans-serif; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: justify; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><strong style="caret-color: rgb(34, 34, 34); color: rgb(34, 34, 34); font-family: &quot;Trebuchet MS&quot;, &quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, &quot;Lucida Sans&quot;, Tahoma, sans-serif; font-size: 14px; font-style: normal; font-variant-caps: normal; letter-spacing: normal; orphans: auto; text-align: justify; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;">Notes -<span class="Apple-converted-space"> </span></strong><span style="caret-color: rgb(34, 34, 34); color: rgb(34, 34, 34); font-family: &quot;Trebuchet MS&quot;, &quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, &quot;Lucida Sans&quot;, Tahoma, sans-serif; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: justify; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration: none; display: inline !important; float: none;">Download counts include these operations performed by other users:</span><ol style="caret-color: rgb(34, 34, 34); color: rgb(34, 34, 34); font-family: &quot;Trebuchet MS&quot;, &quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, &quot;Lucida Sans&quot;, Tahoma, sans-serif; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: justify; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;"><li>Views of your network through the NDEx web app</li><li>Imports to Cyoscape desktop application</li><li>Usage from R, Python or other applications (i.e. IQuery, HiView, etc).</li></ol>

**Comment by Chris-Evelo** [comment]
I wonder whether the high count for Glycolyis is because of the WikiPedia integration. Do we know?

**Reply by AlexanderPico** [reply]
It is unique to this past month and our Wikipedia integration wouldn't point to the NDEx copy of the pathway. I'm guessing this is either due to the new iQuery tool for NDEx or maybe Kristina made a Cytoscape tutorial that pulls this pathway from NDEx?

**Reply by khanspers** [reply]
No new tutorial that uses WP534 from me.

**Comment by Chris-Evelo** [comment]
From: Alexander Pico ***@***.***>
Reply to: wikipathways/wikipathways-development ***@***.***>
Date: Saturday, 5 November 2022 at 20:42
To: wikipathways/wikipathways-development ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [wikipathways/wikipathways-development] NDEx Notifications - Usage statistics of your public networks - October 2022 (Discussion #99)


Dear wikipathways account holder,
Thanks for using NDEx to publish your networks. Here are the usage statistics of your public networks in NDEx for the month of October 2022:

  *   Number of downloads this month: 1530
  *   Total number of downloads: 96692
Most downloaded networks in your account:
Network Name

Downloads

WP534 - Glycolysis and gluconeogenesis - Homo sapiens *

137

WP4172 - PI3K-Akt signaling pathway - Homo sapiens

29

Activating invasion and metastasis

12

WP382 - MAPK signaling pathway - Homo sapiens

11

WP364 - IL-6 signaling pathway - Homo sapiens

10

WP4585 - Cancer immunotherapy by PD-1 blockade - Homo sapiens

10

WP558 - Complement and coagulation cascades - Homo sapiens

10

WP2203 - Thymic stromal lymphopoietin (TSLP) signaling pathway - Homo sapiens

10

WP391 - Mitochondrial gene expression - Homo sapiens

10

P53 pathway

10

WP1449 - Regulation of toll-like receptor signaling pathway - Homo sapiens

9

WP4846 - SARS-CoV-2 and COVID-19 pathway - Homo sapiens

9

WP5095 - Overview of proinflammatory and profibrotic mediators - Homo sapiens

9

NFKB pathway genes with correlation to HOXA1 expression in breast cancer

9

WP4754 - IL-18 signaling pathway - Homo sapiens

9

WP4262 - Breast cancer pathway - Homo sapiens

9

Tumor promoting inflammation

8

Therapeutics

8

WP2855 - Dopaminergic neurogenesis - Homo sapiens

8

Avoiding immune destruction

8


* This network is one of the top 20 downloaded networks in NDEx.
